lib/osv/vulnerability.proto

```diff
@@ -148,14 +134,23 @@ message Affected {
   google.protobuf.Struct database_specific = 5;
 }
 
-// Vulnerability severity information. The exact meaning of these levels are
-// specific to OSS-Fuzz.
-enum Severity {
-  NONE = 0;
-  LOW = 1;
-  MEDIUM = 2;
-  HIGH = 3;
-  CRITICAL = 4;
+message Severity {
+  // Type of the severity.
+  enum Type {
+    UNSPECIFIED = 0;
+    CVSS_V3 = 1;
+  }
+  // The type of this severity entry.
+  Type type = 1;
+  // The quantitative score.
+  string score = 2;
+}
+
+message Credit {
+  // The name to give credit to.
+  string name = 1;
+  // Contact methods.
```

Comment:
should the comment have example of methods ?

Reply:
Clarified this to be URLs. This is all better documented at https://ossf.github.io/osv-schema/ :)